### PR TITLE
icon select

### PR DESCRIPTION
**Pull request by woopy098**
Icon select implemented

### CODE REVIEW

File: src/components/IconOption.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    icon: keyof typeof Icon;
+    color: string;
+    selected: boolean;
+}
+
+const IconOption: React.FC<IconOptionProps> = ({ icon, color, selected }) => {
+    const IconComponent: React.ElementType = Icon[icon];
+
+    return (
+        <div className={`inline-flex justify-center items-center border-4 ${selected ? 'border-blue-500' : 'border-transparent'}`}>
```
Comment (TienDucHo) [inline src/components/IconOption.tsx:16]: Border should be rounded when selected

**Review by TienDucHo — CHANGES_REQUESTED** [review]
Please read my review! Thanks!

File: src/components/IconOption.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// IconOption.tsx
+
+import React from 'react';
+import * as Icon from '@phosphor-icons/react';
+
+interface IconOptionProps {
+    icon: keyof typeof Icon;
+    color: string;
```
[the remark this answers is not in the document]
Comment (woopy098) [reply src/components/IconOption.tsx:8]: color prop change to be className

File: src/components/IconOption.tsx (hunk 1 above)
Comment (woopy098) [reply src/components/IconOption.tsx]: border is rounded

**Review by TienDucHo — APPROVED** [review]

**Pull request merged**